services.md

```diff
@@ -102,6 +102,9 @@ The following services are available on the Yggdrasil v0.4 network, courtesy of
 - [Yggdrasil Forum](http://[21a:34aa:c782:3ad2:1bf8:73f8:141:66e8]) - Forum in yggdrasil, about yggdrasil
   - `http://[21a:34aa:c782:3ad2:1bf8:73f8:141:66e8]`
 
+- [Mesh.Party Lemmy Instance](https://lemmy.mesh.party/) - Yggdrasil and clearnet accessible lemmy instance run by @Stephen304. Requires DNS.
+  - `https://lemmy.mesh.party/`
```

Comment:
Normally we like to have Yggdrasil-specific URLs here, like `http://[201:ad0a:ef6c:3ecd:2928:c59b:7344:196e]/`, so that people can still reach these services if they don't have DNS. Is it possible to make that work?

Reply:
Unfortunately that would mean no host based reverse proxying - should I just remove it then?